diff --git a/node_modules/@ionic/swiftlint-config/index.js b/node_modules/@ionic/swiftlint-config/index.js
index 67d1804..3b9339d 100644
--- a/node_modules/@ionic/swiftlint-config/index.js
+++ b/node_modules/@ionic/swiftlint-config/index.js
@@ -22,4 +22,7 @@ module.exports = {
     'ignores_interpolated_strings': true,
     'ignores_urls': true,
   },
+  identifier_name: {
+    excluded: ['x', 'y', 'i', 'j'],
+  },
 };
